canonical-web-and-design/anbox-cloud.io
# WIP Invitation code

## Done

- [x] Integration with Anbox-cloud API `/login`
- [x] Complete macaroons integration (append to headers)

## QA
- Merge after PR #20 
- Check out this branch
- Run the site using the command `./run serve`
- Go to `/login` which after login will redirect to SSO
- After SSO signed in, it will redirect to `/demo` page and macaroon will be available for use.

## Issue / Card

Fixes #3 

## [Flow diagram](https://docs.google.com/drawings/d/1Hc8lZnMMWyKOVqcBW6rHP2Blfb6ob78GLpbACVKCZ5k/edit)
